src/modules/customers/dto/create-customer.dto.ts

```diff
@@ -25,4 +25,28 @@ export class CreateCustomerDto {
   @MinLength(8)
   @MaxLength(10)
   Company_Registration_Number__c: string;
+
+  @ApiProperty({ description: 'Credit Risk Rating' })
+  @IsString()
+  @IsNotEmpty()
+  CCM_Credit_Risk_Rating__c: string;
+
+  @ApiProperty({ description: 'Credit Risk Rating Date' })
+  @IsString()
```

Comment:
Will this be in `YYYY-MM-DD` format, if yes can we set the length decorator to `10`?

Reply:
Sure, good shout